src/ui/display.c

```diff
@@ -114,6 +113,48 @@ UX_STEP_NOCB(ux_display_confirm_step,
                  "access?",
              });
 
+UX_STEP_NOCB(ux_display_reset_token_signatures_alert,
+             pn,
```

Comment:
To better understand the [layouts](https://developers.ledger.com/docs/nano-app/display-management/#layouts) parameters